README.md

```diff
@@ -37,26 +41,28 @@ module "public_zone" {
     },
   ]
 
-  # If delegation set is null, it will use AWS defaults.
-  # Specify your own nameserver or use an empty list to use AWS defaults.
-  public_secondary_zones = [
+  # Specify your delegated secondary zones (must have their parents defined in 'public_zones')
+  # If delegation set is null, it will use AWS defaults. otherwise specify the delegation set
+  # by reference name as defined in 'delegation_sets' above.
+  # Specify your own name servers or use an empty list to use AWS defaults.
+  public_delegated_secondary_zones = [
     {
       name           = "internal.example.org",
       parent         = "example.org",
       ns_ttl         = 30,
-      nameservers    = [],
+      ns_servers     = [],
```

Comment:
Nameserver servers? Why not leave it as it was?

Reply:
First of all, the TF resource is using `name_servers` instead `nameservers` and I wanted to bring this in sync with `ns_ttl` to highlight similar settings here.

Reply:
WDYT about `ns_list`?

Reply:
That is actually a better fit. Will update this accordingly.